lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -77,25 +78,41 @@ $description = Get-AnsibleParam -obj $params -name "description" -type "str" -de
 $enabled = Get-AnsibleParam -obj $params -name "enabled" -type "bool" -default $true
 $path = Get-AnsibleParam -obj $params -name "path" -type "str" -default '\'
 
+# Ensure the TaskPath starts with and ends with slashes as this causes issues with some
+# versions of Windows
+if ($path[0] -ne '\') {
```

Comment:
@DepthDeluxe I understand, but I don't think we are doing anyone a favor to turn relative paths into absolute paths. We may be misleading them into thinking it is working differently than they are expecting. Better be strict about it and make things clear from that start.